src/modules/media-gallery/components.client.tsx

```diff
@@ -294,10 +294,10 @@ export const MediaGalleryItem = ({
       <AnimatedImage
         // {...handlers}
         className={cn(
-          `object-contain`,
-          !fullscreen ? "w-auto rounded-box" : ` h-full w-full`
+          'object-contain',
+          !fullscreen ? "aspect-square w-auto rounded-box" : ` h-full w-full`
         )}
-        style={{ height }}
+        style={{ height, objectFit: 'cover' }}
```

Comment:
lets not have objectFit cover apply in full screen mode, it creates it so you cant see the full image

![image](https://github.com/jonmumm/KitchenCraft/assets/718391/d0ffa654-997f-4496-98c4-d02dac544e8d)

versus

![image](https://github.com/jonmumm/KitchenCraft/assets/718391/1ebe226b-9c5d-4176-87b1-0a58d99da5fa)